app/models/ncr/work_order.rb

```diff
@@ -26,6 +26,7 @@ def self.purchase_amount_column_name
     before_validation :normalize_values
     before_update :record_changes
 
+    validates :amount, presence: true
```

Comment:
Yeah, I normally don't write specs for "simple" validations for that reason (I would for things like `format`, like below), but am open to adding if you think it's important.